### ISSUE TITLE
Do you use the same subset of VQA for pretraining and finetuning ?

### ISSUE THREAD

**Issue by cdancette** [OFA-Sys/OFA#191]
Hi, I have a question about the pretraining data.

Are you using the same VQA data as for fine-tuning ? I.e. the full VQA train + val (except the 10K validation samples) + VG-QA  ?

Thanks

**Comment by yangapku** [comment]
Hi, for the VQA pretraining subtask, we use the combined dataset from VG-QA, GQA and VQAv2. The details of these dataset parts are given below:
+ The VG-QA part used for pretraining contains more samples than finetuning, since we have also included the samples whose answer is outside the 3,129 most frequent candidate answers. 
+ For the GQA part used for pretraining, we select the samples from the balanced part of the original GQA dataset. 
+ As for the VQAv2 part, compared with finetuning, we only leverage the training split of the original VQAv2 dataset (still keeping the answer restricted in 3,129 most frequent candidate answers, as with the finetuning), excluding samples whose images occur in the validation set of RefCOCO and COCO captioning. 
In total, the VQA pretraining subtask contains around 2.92 millions of QA samples, as mentioned in the OFA paper.

<img width="658" alt="image" src="https://user-images.githubusercontent.com/17445544/182519057-f49690c6-b2f7-47c6-8103-23c0012c8129.png">

**Comment by cdancette** [comment]
@yangapku Thank you very much for the detailed answer.

So If I understood correctly, the pretrained model has never seen any images from the COCO val2014 split ?

**Comment by yangapku** [comment]
For the pretraining datasets possibly related to COCO, we have guaranteed that the pretraining samples do not contain COCO val2014 images.
